main.py

```diff
@@ -275,6 +275,96 @@ def h_rm(args):
 	with open("config.json", "w") as f:
 		f.write(json.dumps(bot.config, sort_keys=True, indent=4))
 
+def h_config(args):
+	# In order to access the config easily
+	bot = halibot.Halibot()
+	bot._load_config()
+
+	# Do we have a module or an agent?
+	if args.destkey:
+		destkey = args.destkey
+	else:
+		is_agent  = args.name in bot.config["agent-instances"]
+		is_module = args.name in bot.config["module-instances"]
+		if is_agent == is_module:
+			if is_agent:
+				print("Both an agent and module exist with that name. -a or -m must be specified.")
```

Comment:
We should probably disallow agents and modules to have the same name, just to prevent some foot-shooting.